File: Reflow/Transformation/Expression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Reflow.Transformation
{
    public class Expression
    {
        //public string Language { get; set; }
        // Hardcode now
        public string Language { get { return "VBScript"; } }
        public string Code { get; set; }
    }
}

File: Java/src/main/java/com/nuarca/etl/transformation/Expression.java
package com.nuarca.etl.transformation;

public class Expression {
    //public string Language { get; set; }
    // Hardcode now
    public String getLanguage() throws Exception {
        return "Java";
    }

    private String _Code;
    public String getCode() {
        return _Code;
    }

    public void setCode(String value) {
        _Code = value;
    }
}
